scripts/meta-transfer-tools.sh

```diff
@@ -0,0 +1,83 @@
+#! /bin/bash
+
+src=
+dst=
+metaDirName=
+configs=
+
+function usage {
+Usage="this tool is a simple wrapper to scp to copy local folder of metad to another (remote)place \n \
+    it afford 4 options:    \n \
+        1. -f (from) local metad dir \n \
+        2. -t (to) remote destination \n \
+        3. -u (update) any configs need to be changed \n \
+            different configs should be seperated by ':' \n \
+            each config has to be the form of "local_ip=172.0.0.1" \n \
+        \n \
+    for example \n \
+        ./meta-transfer-tools.sh -f /path-to-src/metad1 -t bob@172.0.0.1:/path-to-dst -u local_ip=172.0.0.1:port=10086 \n \
+    "
+
+
+echo -e $Usage
+}
+
+while getopts "hf:t:u:" opt; do
```

Comment:
Put a colon in front of `h`, to ignore illegal options

Reply:
OK